indico/htdocs/sass/base/_layout.scss

```diff
@@ -156,3 +156,58 @@ body > * {
 .flex-center {
     justify-content: center;
 }
+
+
+/* Flexbox layout */
+.flexrow {
```

Comment:
I don't think we should provide every class to be orthogonal with the full expressiveness of flex-boxes. flewrow` is just a class for an opinionated layout (flex row non-wrapping).